applications/vane-network.md

```diff
@@ -0,0 +1,317 @@
+# vane-network
+
+
+- **Team Name:** Vane-Network
+- **Payment Address:** 14uNH42NjxS3c4VSxVyHjZYea5ZyJEYWYxjRzmxKQB75pdRB (AssetHub - USDT)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):**  2
+- **Email:** MrishoLukamba@proton.me || abdulrazzaqlukamba@gmail.com ||
+
+
+## Project Overview :page_facing_up:
+
+
+### Overview
+
+- **Beneficiary** Mrisho Lukamba
+  
+
+
+Vane is a parachain that act as an extension product and safety layer for Polkadot and AssetHub which provides services to help trust and growth on social commerce business and introducing transaction safety in web3.
+
+
+Vane-network consist of the following functionalities
+
+ - Confirmation and reversible transactions
+ - Customizable and trustless transactions for social-commerce business
+ - Native token-less parachain enabling usage of DOT, USDT & USDT for all interactions via XCM
```

Comment:
Does this mean your solution enables non-native (e.g. USDT) transfers without the necessity to own the native asset? I believe this isn't included in the milestones yet, could you fix that?